bluesky/protocols.py

```diff
@@ -320,21 +320,19 @@ def describe_collect(self) -> SyncOrAsync[Dict[str, Dict[str, Descriptor]]]:
 class Stageable(Protocol):
     # TODO: we were going to extend these to be able to return plans, what
     # signature should they have?
-    def stage(self) -> List[Any]:
+    def stage(self) -> Status:
```

Comment:
@tacaswell do you think we should make this `Union[Status, List[Any]]` and support both as equally valid, or have the protocols only show the new approach and put the old one in the docs as deprecated?

Reply:
My two cents: I am happy either way on this. I might lean toward annotating both as valid at this early stage.

Reply:
Let's go with the `Union` then